### PR TITLE
new icon: opera (original, original-wordmark)

### PR DESCRIPTION
**Pull request by MichielPater**
- Fix Issue #722

### CODE REVIEW

File: icons/opera/opera-original-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...8 0 0 0 5.07 2.398c3.852 0 6.984-3.199 6.984-7.133s-3.133-7.129-6.984-7.129M64.691 31.926c-4.648 0-8.43-3.793-8.43-8.453s3.781-8.453 8.43-8.453 8.43 3.793 8.43 8.453-3.781 8.453-8.43 8.453m0-18.402c-5.516 0-10.004 4.461-10.004 9.949s4.488 9.949 10.004 9.949 10.004-4.465 10.004-9.949-4.488-9.949-10.004-9.949m48.473 5.312a3.89 3.89 0 0 0-1.609.414c-1.055.535-1.785 1.348-2.254 2.387l-.035.078-.016-2.523h-1.652v13.48h1.656v-.09-6.984c0-.996.164-1.805.508-2.59.316-.73.762-1.379 1.398-1.895a3.21 3.21 0 0 1 2.594-.727l.48.094c.094-.449.32-1.02.414-1.469-.457-.125-1.055-.195-1.484-.176m-14.637 1.509c2.918 0 4.957 1.871 5.301 4.723H93.246c.398-2.668 2.613-4.723 5.281-4.723m0-1.543c-3.852 0-6.984 3.203-6.984 7.133s3.133 7.129 6.984 7.129c2.758 0 5.145-1.633 6.281-4.004l-1.422-.789c-.852 1.918-2.707 3.246-4.859 3.246-2.789 0-5.082-2.238-5.328-5.082h12.297l.02-.5c0-4.203-2.875-7.133-6.988-7.133m22.753 12.695c-2.953 0-5.355-2.508-5.355-5.586s2.402-5.586 5.355-5.586 5.355 2.512 5.355 5.586-2.402 5.586-5.355 5.586m5.07-12.391v2.066a6.5 6.5 0 0 0-5.07-2.391c-3.852 0-6.984 3.199-6.984 7.129s3.133 7.133 6.984 7.133c2.074 0 3.859-.926 5.07-2.398v2.031H128v-13.57zm0 0" fill="rgb(0%,0%,0%)"/></g></svg>
```
Comment (Panquesito7) [inline icons/opera/opera-original-wordmark.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" xmlns:v="https://vecta.io/nano"><defs><linearGradient id="A" gradientUnits="userSpaceOnUse" x1="0" y1="0" x2="1" y2="0" gradientTransform="matrix(0,43.049754,-45.377831,0,19.507578,0.808455)"><stop offset="0" stop-color="rgb(100%,10.588235%,17.647059%)"/><stop offset=".614" stop-color="rgb(100%,10.588235%,17.647059%)"/><stop offset="1" stop-color="rgb(65.490196%,0%,7.843137%)"/></linearGradient><linearGradient id="B" gradientUnits="userSpaceOnUse" x1="0" y1="0" x2="1" y2="0" gradientTransform="matrix(0,38.075183,-40.13424,0,31.26333,3.346278)"><stop offset="0" stop-color="rgb(61.176471%,0%,0%)"/><stop offset=".7" stop-color="rgb(100%,29.411765%,29.411765%)"/></linearGradient></defs><g transform="translate(0 42)"><path d="M23.41.086C10.48.086 0 10.027 0 22.297c0 11.91 9.883 21.629 22.297 22.18.371.016.742.027 1.113.027 5.996 0 11.461-2.137 15.602-5.652a17.54 17.54 0 0 1-9.379 2.723c-5.578 0-10.57-2.625-13.926-6.766-2.59-2.898-4.266-7.184-4.383-11.988V21.77c.117-4.809 1.793-9.094 4.379-11.992 3.359-4.137 8.352-6.758 13.93-6.758 3.43 0 6.637.988 9.383 2.715C34.891 2.238 29.461.109 23.5.086h-.09zm0 0" fill="url(#A)"/><path d="M15.707 9.777c2.148-2.406 4.922-3.859 7.953-3.859C30.477 5.918 36 13.25 36 22.297c0 9.039-5.527 16.371-12.34 16.371-3.031 0-5.805-1.449-7.953-3.855 3.355 4.141 8.348 6.762 13.926 6.762a17.53 17.53 0 0 0 9.379-2.723c4.793-4.066 7.813-9.973 7.813-16.555s-3.02-12.492-7.809-16.559c-2.746-1.73-5.953-2.719-9.383-2.719-5.574 0-10.57 2.621-13.926 6.758" fill="url(#B)"/><path d="M83.078 31.496c-2.953 0-5.355-2.508-5.355-5.586s2.402-5.586 5.355-5.586 5.352 2.512 5.352 5.586-2.402 5.586-5.352 5.586m0-12.715a6.5 6.5 0 0 0-5.07 2.391v-2.066H76.36v17.996h1.648v-6.457a6.48 6.48 0 0 0 5.07 2.398c3.852 0 6.984-3.199 6.984-7.133s-3.133-7.129-6.984-7.129M64.691 31.926c-4.648 0-8.43-3.793-8.43-8.453s3.781-8.453 8.43-8.453 8.43 3.793 8.43 8.453-3.781 8.453-8.43 8.453m0-18.402c-5.516 0-10.004 4.461-10.004 9.949s4.488 9.949 10.004 9.949 10.004-4.465 10.004-9.949-4.488-9.949-10.004-9.949m48.473 5.312a3.89 3.89 0 0 0-1.609.414c-1.055.535-1.785 1.348-2.254 2.387l-.035.078-.016-2.523h-1.652v13.48h1.656v-.09-6.984c0-.996.164-1.805.508-2.59.316-.73.762-1.379 1.398-1.895a3.21 3.21 0 0 1 2.594-.727l.48.094c.094-.449.32-1.02.414-1.469-.457-.125-1.055-.195-1.484-.176m-14.637 1.509c2.918 0 4.957 1.871 5.301 4.723H93.246c.398-2.668 2.613-4.723 5.281-4.723m0-1.543c-3.852 0-6.984 3.203-6.984 7.133s3.133 7.129 6.984 7.129c2.758 0 5.145-1.633 6.281-4.004l-1.422-.789c-.852 1.918-2.707 3.246-4.859 3.246-2.789 0-5.082-2.238-5.328-5.082h12.297l.02-.5c0-4.203-2.875-7.133-6.988-7.133m22.753 12.695c-2.953 0-5.355-2.508-5.355-5.586s2.402-5.586 5.355-5.586 5.355 2.512 5.355 5.586-2.402 5.586-5.355 5.586m5.07-12.391v2.066a6.5 6.5 0 0 0-5.07-2.391c-3.852 0-6.984 3.199-6.984 7.129s3.133 7.133 6.984 7.133c2.074 0 3.859-.926 5.07-2.398v2.031H128v-13.57zm0 0" fill="rgb(0%,0%,0%)"/></g></svg>

```

File: icons/opera/opera-original.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...14" stop-color="rgb(100%,10.588235%,17.647059%)"/><stop offset="1" stop-color="rgb(65.490196%,0%,7.843137%)"/></linearGradient><linearGradient id="B" gradientUnits="userSpaceOnUse" x1="85.463" y1="9.408" x2="85.463" y2="119.121"><stop offset="0" stop-color="rgb(61.176471%,0%,0%)"/><stop offset=".7" stop-color="rgb(100%,29.411765%,29.411765%)"/></linearGradient></defs><path d="M63.996.008C28.652.008 0 28.66 0 64.008c0 34.32 27.02 62.332 60.949 63.922 1.012.047 2.027.074 3.047.074a63.77 63.77 0 0 0 42.652-16.285c-7.5 4.973-16.273 7.836-25.645 7.836-15.242 0-28.891-7.562-38.07-19.484-7.078-8.352-11.66-20.699-11.973-34.559V62.5c.313-13.859 4.895-26.207 11.973-34.559C52.113 16.016 65.762 8.457 81 8.457c9.375 0 18.148 2.863 25.652 7.84C95.383 6.219 80.531.07 64.238.008h-.242zm0 0" fill="url(#A)"/><path d="M42.934 27.945c5.871-6.934 13.457-11.117 21.742-11.117 18.633 0 33.734 21.125 33.734 47.18s-15.102 47.18-33.734 47.18c-8.285 0-15.871-4.18-21.742-11.113 9.18 11.926 22.828 19.484 38.07 19.484 9.375 0 18.145-2.863 25.645-7.836 13.102-11.719 21.348-28.754 21.348-47.715s-8.246-35.988-21.344-47.707c-7.5-4.977-16.273-7.84-25.648-7.84-15.242 0-28.891 7.562-38.07 19.484" fill="url(#B)"/></svg>
```
Comment (Panquesito7) [inline icons/opera/opera-original.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" xmlns:v="https://vecta.io/nano"><defs><linearGradient id="A" gradientUnits="userSpaceOnUse" x1="53.327" y1="2.095" x2="53.327" y2="126.143"><stop offset="0" stop-color="rgb(100%,10.588235%,17.647059%)"/><stop offset=".614" stop-color="rgb(100%,10.588235%,17.647059%)"/><stop offset="1" stop-color="rgb(65.490196%,0%,7.843137%)"/></linearGradient><linearGradient id="B" gradientUnits="userSpaceOnUse" x1="85.463" y1="9.408" x2="85.463" y2="119.121"><stop offset="0" stop-color="rgb(61.176471%,0%,0%)"/><stop offset=".7" stop-color="rgb(100%,29.411765%,29.411765%)"/></linearGradient></defs><path d="M63.996.008C28.652.008 0 28.66 0 64.008c0 34.32 27.02 62.332 60.949 63.922 1.012.047 2.027.074 3.047.074a63.77 63.77 0 0 0 42.652-16.285c-7.5 4.973-16.273 7.836-25.645 7.836-15.242 0-28.891-7.562-38.07-19.484-7.078-8.352-11.66-20.699-11.973-34.559V62.5c.313-13.859 4.895-26.207 11.973-34.559C52.113 16.016 65.762 8.457 81 8.457c9.375 0 18.148 2.863 25.652 7.84C95.383 6.219 80.531.07 64.238.008h-.242zm0 0" fill="url(#A)"/><path d="M42.934 27.945c5.871-6.934 13.457-11.117 21.742-11.117 18.633 0 33.734 21.125 33.734 47.18s-15.102 47.18-33.734 47.18c-8.285 0-15.871-4.18-21.742-11.113 9.18 11.926 22.828 19.484 38.07 19.484 9.375 0 18.145-2.863 25.645-7.836 13.102-11.719 21.348-28.754 21.348-47.715s-8.246-35.988-21.344-47.707c-7.5-4.977-16.273-7.84-25.648-7.84-15.242 0-28.891 7.562-38.07 19.484" fill="url(#B)"/></svg>

```

File: devicon.json [hunk 3]
```diff
@@ -2797,6 +2797,23 @@
...
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#f7192d",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:2816]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
            {
                "base": "original-wordmark",
                "alias": "plain-wordmark"
        ]
```

**Review by Panquesito7 — DISMISSED** [review]

**Comment by Thomas-Boi** [comment]
@amacado @Panquesito7 what do you think about the white lines in the icon? This is caused by the gradient in the `original` version. I'm a bit iffy on this in terms of quality. Should the icon not have that line?

**Comment by Panquesito7** [comment]
> @amacado, @Panquesito7, what do you think about the white lines in the icon? This is caused by the gradient in the `original` version. I'm a bit iffy on this in terms of quality. Should the icon not have that line?

I think that should be removed. If you look at it quickly, it looks fine. But if you look at it carefully, it doesn't look so good.

**Comment by Thomas-Boi** [comment]
> > @amacado, @Panquesito7, what do you think about the white lines in the icon? This is caused by the gradient in the `original` version. I'm a bit iffy on this in terms of quality. Should the icon not have that line?
> 
> I think that should be removed. If you look at it quickly, it looks fine. But if you look at it carefully, it doesn't look so good.

I agree. In that case, @MichielPater can you make a `plain` version please? The `originals` look good so keep both. Just copy and paste `opera-original` to `opera-plain` then remove the gradients. Basically, by the end of it, you should have one continuous red `O`. This should be feasible in an SVG editor. Don't forget to update the `devicon.json` appropriately too (add entry in `font`, remove it in `alias`, etc.)

Extra Credit: Do the same thing for the `original-wordmark` version if you have time. Just make sure that the `plain-wordmark` SVG only contains 1 color => the `Opera` word should also be the same red color as the logo. If you don't have time, just the `plain` version is enough.

File: devicon.json [hunk 4]
```diff
@@ -2880,6 +2880,26 @@
...
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+				"plain",
+				"plain-wordmark"
+            ],
+            "font": [
+                "original",
+				"plain"
```
Comment (Thomas-Boi) [inline devicon.json:2898]:
```suggestion
                "plain",
		"plain-wordmark"
```

**Review by Thomas-Boi — APPROVED** [review]
This looks great @MichielPater!

Good job, I'll merge this PR into `develop`.  

For the other maintainers: don't worry, I checked for whether the SVGs contain strokes. The new SVGs don't have them.

**Pull request merged**